libs/accounts/recovery-phone/src/lib/recovery-phone.manager.ts

```diff
@@ -175,7 +175,7 @@ export class RecoveryPhoneManager {
    * @param uid
    */
   async getAllUnconfirmed(uid: string): Promise<string[]> {
-    const redisKey = `${this.redisPrefix}:${uid}:*`;
+    const redisKey = `*:${this.redisPrefix}:${uid}:*`;
```

Comment:
Not sure this will be efficient. Can we get this work by avoiding a '*' prefix?

Reply:
Looking at `storeUnconfirmed`, I don't understand why the `*` prefix is necessary.